test/e2e/exec_test.go

```diff
@@ -548,4 +549,24 @@ RUN useradd -u 1000 auser`, fedoraMinimal)
 		session.WaitWithDefaultTimeout()
 		Expect(session.OutputToString()).To(Not(ContainSubstring(secretsString)))
 	})
+
+	It("podman exec --wait 2 seconds on bogus container", func() {
+		errString := "Error: timed out waiting for container:"
+		err := errors.New(errString)
+		session := podmanTest.Podman([]string{"exec", "--wait", "2", "1234"})
+		session.WaitWithDefaultTimeout()
+		Expect(session).Should(Exit(125))
+		Expect(err.Error()).To(ContainSubstring(errString))
```

Comment:
this is a NOP, you are never actually checking the command output

you have to use `session.ErrorToString()`